src/components/SQForm/SQFormMultiValue.js

```diff
@@ -0,0 +1,271 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Grid, Chip, TextField, makeStyles} from '@material-ui/core';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import {VariableSizeList} from 'react-window';
+import {useField, useFormikContext} from 'formik';
+import {useForm} from './useForm';
+
+// MUI uses px, a numeric value is needed for calculations
+const LISTBOX_PADDING = 8; // px
+
+const OuterElementContext = React.createContext({});
+
+const OuterElementType = React.forwardRef((props, ref) => {
+  const outerProps = React.useContext(OuterElementContext);
+  return <div ref={ref} {...props} {...outerProps} />;
+});
+
+function renderRow({data, index, style}) {
+  return React.cloneElement(data[index], {
+    style: {
+      ...style,
+      top: style.top + LISTBOX_PADDING
+    }
+  });
+}
+
+// Adapter for react-window
+const ListboxVirtualizedComponent = React.forwardRef(
+  function ListboxVirtualizedComponent(props, ref) {
+    const {children, ...listboxProps} = props;
+    const LIST_MAX_VIEWABLE_ITEMS = 8;
+    const LIST_OVERSCAN_COUNT = 5;
+    const items = React.Children.toArray(children);
+    const ITEM_COUNT = items.length;
+    const ITEM_SIZE = 36;
+
+    const height = React.useMemo(() => {
+      if (ITEM_COUNT > LIST_MAX_VIEWABLE_ITEMS) {
+        return LIST_MAX_VIEWABLE_ITEMS * ITEM_SIZE;
+      }
+      return (
+        items.reduce((acc, _item) => acc + ITEM_SIZE, 0) + 2 * LISTBOX_PADDING
+      );
+    }, [ITEM_COUNT, items]);
+
+    const getItemSize = React.useCallback(() => ITEM_SIZE, []);
```

Comment:
I know this might be copy/pasta from the MUI docs example, but if this only returns `ITEM_SIZE` does this need to be a function?

vs passing `ITEM_SIZE` to the `itemSize` prop on line 59?

Reply:
tbh idk, i copied this from the autocomplete component. i can try to clean it up though.

Reply:
Turns out this does need to be a function, actually. `getItemSize` is being passed to `VariableSizeList` from react-window for the `itemSize` prop which requires a function.